src/gui/foldertreeitemwidget.h

```diff
@@ -77,7 +77,7 @@ class FolderTreeItemWidget : public QTreeWidget {
         static const QColor _sizeTextColor;
         bool _inserting;
         QHash<QString, QTreeWidgetItem *> _subFoldersMap;
-        QHash<QString, QString> _blacklistCache;
+        QHash<QString, QString> _blacklistCache; // nodeId / path
```

Comment:
```suggestion
        using NodeId = QString;
        using Path = QString;
        QHash<NodeId, Path> _blacklistCache;
```
Then re-use `NodeId` and `Path` in  the class wherever they can clarify something.

Reply:
Not a fan to use `NodeId` since it is already a used type in the project.
What do you think about fd5879180005c11782f9c30413635a4dd43451dc